pyaedt/modules/Boundary.py

```diff
@@ -803,12 +1001,42 @@ def identify_extent_poly(self):
             return False
         return outlines[0].name
 
+    @property
+    def board_cutout_material(self):
+        """Material applied to cutouts regions."""
+        return self.props["NativeComponentDefinitionProvider"].get("BoardCutoutMaterial", "air ")
+
+    @property
+    def via_holes_material(self):
+        """Material applied to via holes regions."""
+        return self.props["NativeComponentDefinitionProvider"].get("ViaHoleMaterial", "copper")
+
+    @board_cutout_material.setter
+    def board_cutout_material(self, value):
+        """Set material to apply to cutouts regions.
```

Comment:
The "s" in "cutouts" is odd here. Can the "s" be removed in the method description and the parameter description?